app/case/forms.py

```diff
@@ -38,20 +37,6 @@ def save(self, *args, **kwargs):
         return user
 
 
```

Comment:
cleaning up some old weird experimental stuff (DynamicTableForm) that I can finally delete